Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MediaPlayer.java

```diff
@@ -204,7 +204,7 @@ public void addMediaFile(String viewID, String source, String actionPlayerID, St
                     audioPlayer.setTimerColorAndSize(Color.GRAY, 14);
                 }
             } else {
-                audioPlayer = new MediaPlayerView(context, activity);
+                audioPlayer = new MediaPlayerView(context, activity, false);
```

Comment:
make it configurable. Don't hardcode it here, else in future it would be one more apk update to make this change.